code/DDSCodeTester.cpp

```diff
@@ -1185,6 +1186,34 @@ void dds_discovery_examples()
             return;
         }
         //!--
+
+        // Check XML static discovery from file
+        std::string file = "file://static_Discovery.xml";// The (file://) flag it is optional.
+        DomainParticipantFactory* factory = DomainParticipantFactory::get_instance();
+        if (factory->check_xml_static_discovery(file) != ReturnCode_t::RETCODE_OK)
+        {
+            std::cout << "Error parsing xml file " << file << std::endl;
+        }
+        //!--
+
+        // Check XML static discovery from data
```

Comment:
Maybe a comment stating that data:// is required for this specific case would be helpful for the users.